docs/docs/backoff.md

```diff
@@ -0,0 +1,51 @@
+---
+title: No new instances of app are starting, it is in 'delayed' mode
+---
+
+# No new instances of app are starting, it is in 'delayed' mode
+
+One of the reasons why is your deployment not finishing and your instances appear to start very slowly is that your application 
+might be in 'delayed' mode.
```

Comment:
there are many line breaks throughout.. these should just be one line IMO

Reply:
I submit we should do a sentence per line as our general standard; I really like how well diffs render.